usabilla/stylabilla
# Make mood badges consistent with the other badges

Jira: https://usabilla.atlassian.net/browse/BLUP-865
Preview: https://static-staging.usabilla.com/stylabilla/pr/fix-mood-sizes/docs/section-components.html#kssref-components-badge-mood

## What to test? Scope of the test?

- Use MS function to calculate badges sizes.
- The mood badges should be consistent with the other badges. SO i added another class `sb-mood-- badge` which does the magic.
- Add `xxs` size for moods, it can be used in the Mood Chart tooltip:
<img width="248" alt="Screenshot 2019-07-04 at 14 14 25" src="https://user-images.githubusercontent.com/10066634/60673157-6be55800-9e77-11e9-96cd-35fc5c9cdc8b.png">

- Fix mood_rating preview, using default size for mood icons.

## Reviews

Reviewers: @chiel @MLKiiwy @thefrontendwizard @kagnamazian @realdreamer @moroale93 @DavideBruner @ranst91-u @galagmon @Maruceau 

Required: 1 FE approval - 1 UI approval

## Checklist minimal
- [ ] Export the component by importing it in the index.scss
- [ ] Components are navigable by keyboard only
- [ ] [Accessibility (ARIA attributes)](https://w3c.github.io/using-aria/) are included
- [ ] Semantic HTML is used
- [ ] The component works across all device widths, and is built mobile first.
- [ ] Example for each modifier with description
- [X] Example for each component with description

See [definition of done](https://github.com/usabilla/stylabilla/blob/master/CONTRIBUTING.md#component-definition-of-done)

## Browsers compatibility checklist
These browsers were manually tested:
- [X] Chrome last version
- [ ] Firefox last version
- [ ] Edge last version
- [X] Safari 10+
- [ ] IE9 - IE10 - Only if your element can be part of a public feedback form
